src/createServer.js

```diff
@@ -1,11 +1,23 @@
-'use strict';
+'use strict'; // eliminates silent errors, works with defaults and more
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { userRouter } = require('./routes/userRoute');
+const { expenseRouter } = require('./routes/expenseRoute');
+const userService = require('./services/userService');
+const expenseService = require('./services/expensesService');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+  userService.start();
+  expenseService.start();
```

Comment:
The `start` methods in `userService` and `expenseService` are called synchronously. If these methods involve database or I/O operations, they should be made asynchronous to prevent blocking the event loop.